app/src/main/java/com/duckduckgo/app/notification/AndroidNotificationScheduler.kt

```diff
@@ -84,22 +71,10 @@ class NotificationScheduler(
         workManager.cancelAllWorkByTag(UNUSED_APP_WORK_REQUEST_TAG)
 
         when {
-            variant().hasFeature(Day1DripA1Notification) && dripA1Notification.canShow() -> {
```

Comment:
This code comes from the notification drip experiment. I've clean it up because is not active anymore. There is a task in that project to remove all the code that's not needed anymore but doing this here means better readability for this new experiment/